README.md

```diff
@@ -1,76 +1,89 @@
 # Notification service
```

Comment:
Update the title to "Central Event Processor Service"

Reply:
Title updated.